### ISSUE TITLE
Measure the impact of the project in the Chamber of Deputies

### ISSUE THREAD

**Issue by Irio** [datasciencebr/serenata-de-amor#153]
As important as finding corruption in the government, there's the problem of measuring the impact Serenata de Amor is having on making people commit less actions of corruption.

Did we have a decrease of suspect acts since the project started to get known? In layman's terms, suspect reimbursements decreased in amount and value after Serenata started to get known in the media?

Suggested by @joaomalossi.

**Comment by joaomalossi** [comment]
Also we can use this model to indicate suspicious behavior by parliamentarians that got worried after the project started working and became restrained in their expenses.

**Comment by cuducos** [comment]
Now we have a good window to look back and compare, for instance 2017 with 2016 — just remember to take the 90 days interval in which congresspeople can claim for reimbursements into account.

**Comment by kurtmaia** [comment]
Have any analysis on this come out?

**Comment by cuducos** [comment]
> Have any analysis on this come out?

[Yep](https://medium.com/serenata/o-impacto-do-controle-social-na-c%C3%A2mara-dos-deputados-c2b2a34db09e) : )

Sure thing this is not _the definitive analysis_. But it's one possible take on it, @kurtmaia.

**Comment by kurtmaia** [comment]
> Sure thing this is not the definitive analysis. But it's one possible take on it, @kurtmaia.

Oh that's nice. Thanks for sharing that.  I work with statistical network analysis, I was considering an analysis of whole "gastos" network using some dynamic networks methods. The idea is see if there is an overall difference of behavior of the network before and after serenata.
